README.md

```diff
@@ -1,2 +1,59 @@
 # OctolinerPi
-Octoliner library for Raspberry Pi
+
+A library for Raspberry Pi to interface with the Amperka [Octoliner](https://my.amperka.com/modules/octoliner) 8-channel line sensor.
+
+## Installation
+
+Open the terminal on your Raspberry Pi and use `pip` to install the library:
+
+```shell
+pip3 install octoliner
+```
+
+If you haven’t enabled I²C support in your Raspbian Linux yet, run the configuration tool to turn it on:
+
+```shell
+sudo raspi-config
+```
+
+Then: Interfacing Options → I2C → Yes (enable) → Yes (autoload) → \<Finish\> → Yes (reboot). The setting preserves across reboots.
+
+```console
+$ python3 -m octoliner
```

Comment:
Maybe you should add the subtitle “Usage” or something like that before `python3 -m octoliner`? All the same, this example does not apply to the Installation section.